tests/system/test_integration.py

```diff
@@ -102,6 +102,20 @@ def test_load_docs_with_template_and_add_error(self):
 
         self.check_backend_error_sourcemap(self.index_error, count=4)
 
+    def test_jaeger_http(self):
```

Comment:
What do you think about creating a dedicated `test_jaeger.py` file where we bundle all jaeger related tests? (Similar to how we separate jaeger from the Intake API in the go code).

Reply:
Yeah I was just being lazy, I'll do that.